libs/design-system/src/lib/Components/LandingHeader/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ButtonLink } from '../Button';
+import { GradientName, LandingHeaderContainer } from './LandingHeaderContainer';
+
+export type LandingHeaderProps = {
+  title: string;
+  gradient: GradientName;
+  callout: string;
+  description: string;
+  buttonText: string;
+  buttonUrl: string;
+};
+
+export function LandingHeader({
+  buttonText,
+  buttonUrl,
+  callout,
+  description,
+  gradient,
+  title,
+}: LandingHeaderProps) {
+  return (
+    <LandingHeaderContainer gradient={gradient}>
+      <div className="flex max-w-6xl flex-col justify-items-stretch px-4 py-12 md:px-20 md:py-20">
```

Comment:
I just pushed a container tailwind config to the branch. Removing the max-w-6xl and wrapping this with a `<div className="container"` will center everything and make the max width 1140px